src/api-reference/request/v4.request.markdown

```diff
@@ -88,6 +84,8 @@ Name|Type|Format|Description
 ---|---|---|---
 -|object|[Request](#request)|The Request having {requestUuid} as unique identifer
 
+[Back to top](#request-endpoints)
```

Comment:
Let's get rid of these 'Back to top' links -- they unnecessarily complicate the documentation and with anchor points the back button in the browser serves this purpose and these tend to interfere with that muscle memory as well.

Reply:
Ok

Reply:
Hi Charlie,
I got this idea from the Receipt Endpoing available in LiveSite from which we can take example from (https://developer.concur.com/api-reference/receipts/endpoints.html) and I feel that from an end-user experience using our documentation, this will avoid quantities of scrolling and make them access quicker to the required information.
Is this a blocking issue? Because I really would like to maintain those as from my perspective, I've been reviewing the documentation with and without those Back to top, and talking around me this is an added value to end-users. 
Thanks